providers/snowflake/role_grant.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2019 The Terraformer Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package snowflake
+
+import (
+	"fmt"
+
+	"github.com/GoogleCloudPlatform/terraformer/terraform_utils"
+)
+
+type RoleGrantGenerator struct {
+	SnowflakeService
+}
+
+func (g RoleGrantGenerator) createResources(roleGrantList []roleGrant) []terraform_utils.Resource {
+	type tfGrant struct {
+		Role      string
+		Privilege string
+		Roles     []string
+		Shares    []string
+	}
+	groupedResources := map[string]*tfGrant{}
+	for _, grant := range roleGrantList {
+		// TODO(ad): Fix this csv delimited when fixed in the provider. We should use the same functionality.
+		id := grant.Name.String
```

Comment:
these probably have a different ID

Reply:
https://github.com/chanzuckerberg/terraform-provider-snowflake/blob/master/pkg/resources/role_grants.go#L73

Reply:
oh, you're right!